akvo/rest/views/indicator.py

```diff
@@ -12,7 +12,10 @@
 class IndicatorViewSet(PublicProjectViewSet):
     """
     """
-    queryset = Indicator.objects.all()
+    queryset = Indicator.objects.all().select_related(
```

Comment:
Yes, I realized I should've done this just after I asked you to look at it. Fixed it now.